orderer/consensus/etcdraft/util_test.go

```diff
@@ -38,6 +40,16 @@ func TestIsConsenterOfChannel(t *testing.T) {
 		"ZJQ0FURS0tLS0tCg==")
 	require.NoError(t, err)
 
+	someRandomCert, err := base64.StdEncoding.DecodeString("LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJxakNDQVZHZ0F3SU" +
```

Comment:
Same comment here for how to generate this cert.  It seems like we really mostly don't care about the contents, as implied by the name, but it very likely will not be clear to the next reader.

Reply:
I removed it.